File: Law App/LawApi/Database/AnnouncingMoreRealEstate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace LawApi.Database
{
    [Table("AnnouncingMoreRealEstate")]
    public class AnnouncingMoreRealEstate
    {
        public int ID { get; set; }
        public string Lawname { get; set; }
        public string Type { get; set; }
        public string Owners { get; set; }
        public string Address { get; set; }
        public string Space { get; set; }
        public string Price { get; set; }
        public string Date { get; set; }
        public string Dscrp { get; set; }
        public string Img { get; set; }
        public int Status { get; set; }         
    }
}

File: Law App/law/lib/class/announcingmorerealestate.dart
class AnnouncingMoreRealEstateData {
  int? id;
  String? lawname;
  String? type;
  String? owners;
  String? space;
  String? address;
  String? price;
  String? date;
  String? dscrp;
  String? img;
  int? status;

  AnnouncingMoreRealEstateData(
      {this.id,
      this.lawname,
      this.type,
      this.owners,
      this.space,
      this.address,
      this.price,
      this.date,
      this.dscrp,
      this.img,
      this.status});

  AnnouncingMoreRealEstateData.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    lawname = json['lawname'];
    type = json['type'];
    owners = json['owners'];
    space = json['space'];
    address = json['address'];
    price = json['price'];
    date = json['date'];
    dscrp = json['dscrp'];
    img = json['img'];
    status = json['status'];
  }
  static List<AnnouncingMoreRealEstateData> fromJsonList(
      List<dynamic> jsonList) {
    return jsonList
        .map((json) => AnnouncingMoreRealEstateData.fromJson(json))
        .toList();
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['lawname'] = lawname;
    data['type'] = type;
    data['owners'] = owners;
    data['space'] = space;
    data['address'] = address;
    data['price'] = price;
    data['date'] = date;
    data['dscrp'] = dscrp;
    data['img'] = img;
    data['status'] = status;
    return data;
  }
}
